### PR TITLE
part of #1804, polish documents

### PR DESCRIPTION
**Pull request by manuzhang**

### CODE REVIEW

File: docs/commandline.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ storm -verbose -config storm.yaml -jar storm-starter-${STORM_VERSION}.jar storm.
 To start a Gearpump Cluster on YARN, you can:
 
 ```bash
-yarnclient launch -package /usr/lib/gearpump/gearpump-2.11-{{ site.GEARPUMP_VERSION }}.zip
+yarnclient launch -package /usr/lib/gearpump/gearpump-{{ site.SCALA_BINARY_VERSION }}-{{ site.GEARPUMP_VERSION }}.zip
```
Comment (clockfly) [inline docs/commandline.md]: Sure?

Comment (manuzhang) [inline docs/commandline.md]: snip20151229_1

**Comment by clockfly** [comment]
+1

**Comment by clockfly** [comment]
Please update the website by following guide in https://github.com/gearpump/gearpump.github.io

**Pull request merged**